src/input_utils.py

```diff
@@ -17,13 +17,15 @@
 import attr
 import tensorflow as tf
 import tensorflow_text as tf_text
+from einops import reduce, rearrange
 from official.modeling import tf_utils
+from tensorflow.python.ops import math_ops
```

Comment:
Nit: not sorted alphabetically.  (Ignore the leading "import"/"from" when sorting.)